contracts/multisig-prover/src/state.rs

```diff
@@ -24,3 +28,42 @@ pub const COMMANDS_BATCH: Map<&BatchID, CommandBatch> = Map::new("command_batch"
 pub const MULTISIG_SESSION_BATCH: Map<u64, BatchID> = Map::new("multisig_session_batch");
 
 pub const REPLY_BATCH: Item<BatchID> = Item::new("reply_tracker");
+
+#[cw_serde]
+pub struct WorkerSet {
+    pub signers: Vec<Signer>,
+    pub threshold: Uint256,
```

Comment:
which threshold is this?

Reply:
signing threshold. Defines how many signatures are needed before relaying to the external gateway